src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -281,11 +276,7 @@ const ConfigureEmbedding = (props) => {
 
     const plotActions = {
       export: true,
-      source: false,
-      compiled: false,
-      editor: false,
```

Comment:
We have various implementations of this. Some plots only show the Export options. Some show all the options. We need to refactor this and make it uniform for all plots. This might be appropriate together with the `scaleFactor` changes.

Reply:
New ticket created for this: [https://biomage.atlassian.net/browse/BIOMAGE-1782](https://biomage.atlassian.net/browse/BIOMAGE-1782)